cmd.go

```diff
@@ -306,6 +307,12 @@ func RunCommandLineWithOptions() error {
 
 					cbytes, err := Codify(os.Stdin, codifyValues)
 					if err != nil {
+						if len(cbytes) > 0 {
+							// We have both
+							fmt.Println(string(cbytes))
+							fmt.Fprintf(os.Stderr, "\nWARNING ⚠\n\nUnable to parse full system: %v\n\nWARNING ⚠\n", err.Error())
```

Comment:
I thought this should go [here](https://github.com/kris-nova/naml/blob/main/codify.go#L380) so the error message can be explicit about the Kind that couldn't be converted. That sort of specificity in error messages is helpful.